addons/website/tests/test_snippets.py

```diff
@@ -53,3 +53,6 @@ def test_06_snippet_image_gallery(self):
             'url': base + '/web/image/website.s_banner_default_image.jpg',
         })
         self.start_tour("/", "snippet_image_gallery", login='admin')
+
+    def test_07_snippet_images_wall(self):
+        self.start_tour("/", "snippet_images_wall", login='admin')
```

Comment:
Quote consistency, even if the rest of the file is mix of single and double, I think that in python we encourage single quote string